pkg/api/unversioned/deep_copy_generated.go

```diff
@@ -21,127 +21,15 @@ limitations under the License.
 package unversioned
 
 import (
-	"time"
-
 	conversion "k8s.io/kubernetes/pkg/conversion"
+	time "time"
 )
 
-func DeepCopy_unversioned_APIGroup(in APIGroup, out *APIGroup, c *conversion.Cloner) error {
```

Comment:
hmm - actually I've just started wondering why there were exposed before my change and now they are no longer exposed. And to be honest, now I don't understand why.
[Those types comes from k8s.io/kubernetes and all their fields also...]
Any hints?